shared-libs/transitions/src/lib/messages.js

```diff
@@ -1,6 +1,6 @@
 const _ = require('underscore'),
   phoneNumber = require('@medic/phone-number'),
-  messageUtils = require('@medic/message-utils'),
+  messageUtils = require('@garethbowen/message-utils'),
```

Comment:
I can't add a comment that far down, but in `addMessage` function, the default recipient is set to `clinic`.
Maybe we should change it to `parent`. I'm pretty sure at least some messages don't specify recipients (fe `default_responses`)
Also, `addErrors` fn also sends messages to `clinic`.

Reply:
I've changed both to "reporting_unit". This means it should be sent to the report's `from` property. Both addMessage and addErrors should report back to the person who sent the report so I think it makes sense...